zookeeper-metrics-providers/zookeeper-prometheus-metrics/src/test/java/org/apache/zookeeper/metrics/prometheus/PrometheusMetricsProviderTest.java

```diff
@@ -56,6 +56,7 @@ public void setup() throws Exception {
         CollectorRegistry.defaultRegistry.clear();
         provider = new PrometheusMetricsProvider();
         Properties configuration = new Properties();
+        configuration.setProperty("httpHost", "127.0.0.1"); // local host for test
         configuration.setProperty("httpPort", "0"); // ephemeral port
```

Comment:
PortAssignment.unique()?

Reply:
I think the port 0 is enough to covert this PR.
By the way, try to import class `PortAssignment` which is a test class in `zookeeper-server` module, can't imported easily.